src/ECS/utilities/Zipper/GenericZipper.hpp

```diff
@@ -0,0 +1,67 @@
+/*
+** EPITECH PROJECT, 2024
+** RType
+** File description:
+** GenericZipper.hpp file
+*/
+
+#ifndef GENERIC_ZIPPER_HPP
+    #define GENERIC_ZIPPER_HPP
+
+    #include <algorithm>
+    #include <cstddef>
+    #include <vector>
+
+template<class ZipIt, class... Containers>
+class GenericZipper {
```

Comment:
Could you add a doxygen comment for this class ? 

For a class, you can add the @class keyword with the name of the class and a @brief description resuming the main goal.